lib/rc/rc.go

```diff
@@ -232,7 +232,7 @@ func (p *Process) Events(since int) ([]Event, error) {
 	dec.UseNumber()
 	err = dec.Decode(&evs)
 	if err != nil {
-		return nil, fmt.Errorf("Events: %s in %q", err, bs)
+		return nil, fmt.Errorf("events: %s in %q", err, bs)
```

Comment:
%w?

Reply:
Requires turning the error around, %w only works as the last verb. This is internal so I didn't bother.

Reply:
"only works with last verb" lol wat.

Reply:
It used to be:

> If the last argument is an error and the format string ends with ": %w", the returned error implements an Unwrap method returning it. (https://godoc.org/golang.org/x/xerrors#Errorf)

apparently that was relaxed in the final implementation